### DISCUSSION TITLE
[Feature] Share recognized people list to search photos

### DISCUSSION THREAD

**Discussion by losysh** [immich-app/immich#8489] [Feature Request]
### The feature

We have huge family photo archive. All photos are loaded into Immich as external libraries (separated by years). External libraries were analyzed and got people list. All photos belong to one user. To access to archive shared albums are created. So every user can see all photos, but only owner can search photos by person imaged on photo. It's good to have possibility to use shared people list and look for photos by person for every user who have access to album.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5089
